bugbug/bug_features.py

```diff
@@ -185,11 +185,15 @@ def cleanup_url(text):
     return re.sub(r'http\S+', 'URL', text)
 
 
+def cleanup_fileref(text):
+    return re.sub(r'\w+\.py\s?|\w+\.json\s?|\w+\.js\s?|\w+\.jsm\s?|\w+\.html\s?|\w+\.css\s?|\w+\.c\s?|\w+\.cpp\s?|\w+\.h\s?', 'FILE_REFERENCE ', text)
```

Comment:
Using `\s?` makes this match a lot of things that shouldn't be matched.
E.g. any string that contains '.c' will be matched. Instead, we should only match strings which end with `.c`.

Maybe just replacing `\s?` with `\b` and removing the whitespace at the end of `FILE_REFERENCE` will be enough.